docs/mino.md

```diff
@@ -127,3 +131,53 @@ when necessary. For instance, for a tree-based algorithm, it *could* look like:
 A message coming from F would then be relayed through B and A to reach the right
 side of the tree. This kind of algorithm is efficient in terms of distributed
 load but is very sensible to failures so this is of course only an example.
+
+#### Example
+
+The following illustrates how to use the stream API to send a message that a
+node will simply send back.
```

Comment:
Feel free to ignore this, but this may be more immediately obvious.

```suggestion
This illustrates how to use the stream API to implement a simple ping service, 
where a message is echoed back to the node who sent it.
```